bin/expose.js

```diff
@@ -1,3 +1,168 @@
 #!/usr/bin/env node
 
-console.log("expose");
+const yargs = require("yargs/yargs");
+const { hideBin } = require("yargs/helpers");
+
+const startLocalServer = require("../lib/start_local_server");
+const watchFileSystem = require("../lib/watch_file_system");
+const syncFiles = require("../lib/utils/sync_files");
+const stringify = require("../lib/utils/stringify");
+
+const ApiClient = require("../lib/api_client");
+
+const { API_ENDPOINT } = require("../lib/constants");
+
+const {
+  log,
+  logFatal,
+  logUpload,
+  logInfo,
+  wrapRed,
+  logHelp
+} = require("../lib/utils/log");
+
+const argv = yargs(hideBin(process.argv))
+  .options({
+    "token": {
+      alias: "a",
+      describe: "Your Eventmaker auth token to access the REST API",
+      demandOption: true
+    },
+    "eventId": {
+      alias: "e",
+      describe: "The Eventmaker event id you are working on",
+      demandOption: true
+    },
+    "initialSync": {
+      alias: "s",
+      describe: "Wether or not the theme will be entirely synced on launch",
```

Comment:
You mean `Whether` not `wether` ([mouton](https://www.wordreference.com/enfr/wether)) ?